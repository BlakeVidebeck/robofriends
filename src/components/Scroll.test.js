import { shallow } from 'enzyme';
import Scroll from './Scroll';

it('Scroll renders correctly', () => {
	const wrapper = shallow(<Scroll />);
	expect(wrapper).toMatchSnapshot();
});
