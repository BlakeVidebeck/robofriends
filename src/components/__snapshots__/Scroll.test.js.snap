// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Scroll renders correctly 1`] = `
<div
  style={
    Object {
      "border": "1px solid black",
      "height": "800px",
      "overflowY": "scroll",
    }
  }
/>
`;
